src/main/java/jpos/config/simple/xml/JavaxRegPopulator.java

```diff
@@ -0,0 +1,450 @@
+                        Object obj = null;
+                        obj = JposEntryUtility.parsePropValue(attrs.getValue("value"), type);
+                        currentEntry.addProperty(attrs.getValue("name"), obj);
+                    } catch (Exception e) {
+                        currentEntry = null;
+                        String msg = "Invalid prop: name=" + attrs.getValue("name")
+                                + ":value=" + attrs.getValue("value");
+                        tracer.println(": " + msg);
+                        lastLoadException = new SAXException(msg, e);
+                    }
+                    ;
+                }
+            }
+            tracer.println("");
+        }
+
+        private void addOptionalProperty(String name, String value) {
+            currentEntry.addProperty(name, value == null ? "" : value);
+        }
+
+        @Override
+        public void endElement(String uri, String lname, String qname) {
+            if (theException == null)
+                tracer.println("EndElement: " + qname);
+            else {
+                tracer.println("EndElement: " + theException.getMessage());
+                theException = null;
+            }
+            if (qname.equals("JposEntry")) {
+                if (currentEntry != null)
+                    jposEntryList.add(currentEntry);
+                currentEntry = null;
+                theException = null;
+            }
+        }
+
+        //----------------------------------------------------------------
+        // ErrorHandler
+
+        @Override
+        public void error(SAXParseException e) {
+            currentEntry = null;
+            theException = e;
+        }
+
+        @Override
+        public void fatalError(SAXParseException e) {
+            currentEntry = null;
+            theException = e;
+        }
+
+        //----------------------------------------------------------------
+        // EntityResolver
+
+        @Override
+        public InputSource resolveEntity(String name, String publicId, String uri, String systemId) {
+            tracer.println("JposEntityResolver:resolveEntity:publicId=" +
+                    publicId);
+
+            tracer.println("JposEntityResolver:resolveEntity:systemId=" +
+                    systemId);
+
+            if (publicId.equals(DTD_DOC_TYPE_VALUE)) {
+                InputStream is =
+                        getClass().getResourceAsStream(DTD_FILE_NAME);
+
+                if (is == null)
+                    is = findFileInClasspath(DTD_FILE_NAME);
+
+                if (is != null)
+                    return new InputSource(new InputStreamReader(is));
+            }
+
+            return null;
+        }
+    }
+
+    //--------------------------------------------------------------------------
+    // Instance variables
+    //
+
+    private List<JposEntry> jposEntryList = new LinkedList<>();
```

Comment:
Forgot one thing.
The original implementor used LinkedList quite often. 
However, we do not sort the list and do not insert. So, does not make sense to use the more sophisticated `LinkedList`. An `ArrayList`will do it too. 
```suggestion
    private List jposEntryList<JposEntry> = new ArrayList<>();
```

Reply:
... Sorting took place afterwards, so I guess LinkedList is OK here.